frontend/src/metabase/query_builder/components/expressions/Editor/Editor.tsx

```diff
@@ -4,48 +4,50 @@ import cx from "classnames";
 import { useCallback, useMemo, useRef, useState } from "react";
 import { useMount } from "react-use";
 import { t } from "ttag";
+import _ from "underscore";
 
 import { useSelector } from "metabase/lib/redux";
 import { getMetadata } from "metabase/selectors/metadata";
-import { Box } from "metabase/ui";
+import { Box, Button, Flex, Icon } from "metabase/ui";
```

Comment:
The help popover covers the error message, and it just really feels like too many popovers.

![image](https://github.com/user-attachments/assets/73c77ed4-8c01-4c07-bd49-775e56848ac9)

Reply:
I finished typing and I can't click "Done" because it's covered by the help popover

![image](https://github.com/user-attachments/assets/b31b33bd-f01a-46c0-a6c8-e81e04261f43)

Reply:
Agreed. @AlessioLaiso What do you think?